.github/workflows/ci-dgraph-ldbc-tests.yml

```diff
@@ -0,0 +1,71 @@
+name: ci-dgraph-ldbc-tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "30 * * * *"
+jobs:
+  dgraph-load-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout LDBC query repo
+        uses: actions/checkout@v3
+        with:
+          repository: dgraph-io/ldbc
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: ldbc
+      - uses: actions/checkout@v3
```

Comment:
If I understand correctly this means we will check out the ldbc repo _nested_ inside the dgraph repo, correct?  In that case maybe we can make this more clear by first checking out Dgraph, then ldbc:
```
- name: Checkout Dgraph
  uses: actions/checkout@v3
- name: Checkout LDBC Query Repo
  uses: actions/checkout@v3
  with:  
         ....
```